### ISSUE TITLE
Geographic Coordinate System

### ISSUE THREAD

**Issue by ndow-user** [oss-wec/wisdm#35]
## I'm submitting a ...

**put an x between the brackets to check the box**

  - [ ] bug report - something didn't work as expected
  - [ ] feature request - the app doesn't do something it should
  - [x] support request - any questions or comment

## What browser are you using?

* [x] Chrome
* [ ] Firefox
* [ ] Edge
* [ ] Internet Explorer (check if the issue exists on a different browser, Explorer isn't a supported browser)
* [ ] Other, please specify:

## What is the current behavior?
 When entering a new animal encounter the default GCS is in Lat Long. Shouldn't this be in UTMs given everyone in the department and at the state government level uses the UTM coordinate system? 
 

## Please explain in detail the bug, feature, or question:

**Comment by kissmygritts** [comment]
Mapping data on the web almost always uses WGS83 (a projection called web mercator). I do have plans on allowing users to select the either, then having PostGIS do the conversion. However, I haven't integrated PostGIS yet, so this is temporary.

**Comment by ndow-user** [comment]
I agree, it would be best to create a GCS conversion to allow users to enter coordinates from the projection of their choosing. In the past, I've dealt with websites that only accept NAD 83 or WGS 84 but in degrees, minutes, seconds. This put the burden of converting the coordinates on the data technician and ultimately slowed down their productivity and opened up the georeferencing to user error when making those conversions. There are also many open-source conversion programs available that make it difficult for standardization of this process.
